### PR TITLE
gpxlab: fix localization

### PR DESCRIPTION
**Pull request by sikmir**
###### Motivation for this change
https://github.com/NixOS/nixpkgs/issues/86054

###### Things done
- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Added a release notes entry if the change is major or breaking
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 124012` at b9315402 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built successfully:</summary>
  <ul>
    <li>gpxlab</li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/unclear-gpl.md">unclear-gpl</a>

`gpl3` is a deprecated license, please check if project uses `gpl3Plus` or `gpl3Only` and change `meta.license` accordingly.

Near pkgs/applications/misc/gpxlab/default.nix:41:5:
```
   |
41 |     license = licenses.gpl3;
   |     ^
```
</li>
  </ul>
</details>

**Comment by symphorien** [comment]
Mmmh how could this break aarch64 build?

**Comment by symphorien** [comment]
@GrahamcOfBorg build gpxlab

File: pkgs/applications/misc/gpxlab/default.nix [hunk 1]
```diff
@@ -24,16 +31,14 @@ mkDerivation rec {
...
     homepage = "https://github.com/BourgeoisLab/GPXLab";
     description = "Program to show and manipulate GPS tracks";
     longDescription = ''
       GPXLab is an application to display and manage GPS tracks
       previously recorded with a GPS tracker.
     '';
-    license = licenses.gpl3;
+    license = licenses.gpl3Only;
     maintainers = with maintainers; [ sikmir ];
     platforms = with platforms; linux ++ darwin;
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/gpxlab/default.nix:43]:
```suggestion
    platforms = platforms.unix;
```

**Pull request merged**